doc/index.html

```diff
@@ -590,11 +572,7 @@ <h3 id="command-line-options">Command-line options</h3>
 
  Note that if you use this option, Randoop is nondeterministic: it
  may generate different test suites on different runs. [default 100]</li>
-      <li id="option:outputlimit"><b>--outputlimit=</b><i>int</i>. Determines the maximum number of tests to output.
- 
- This command-line option is generally more appropriate than
- --inputlimit, which controls how many test candidates Randoop
- generates internally. [default 100000000]</li>
+      <li id="option:outputlimit"><b>--outputlimit=</b><i>int</i>. Determines the maximum number of tests to output. [default 100000000]</li>
```

Comment:
For consistency with following line (and to avoid excess verbiage), cut "Determines the".  Also, it's OK to remove the guidance about what option to use, but what replaces it?  Randoop has a lot of command-line options and a user may feel overwhelmed; how should a user decide which ones to use?